app/views/shared/_application_form.erb

```diff
@@ -0,0 +1,5 @@
+<h1 class="govuk-heading-m">Application form</h1>
+<p class="govuk-hint">Do we accept applications from this country?</p>
+<%= f.govuk_radio_button :eligibility_enabled, 'yes', label: { text: 'Yes' }, link_errors: true %>
+<%= f.govuk_radio_button :eligibility_enabled, 'no', label: { text: 'No' } %>
+<br>
```

Comment:
`<br>` should be avoided when there only for layouts because it poses accessibility issues. I think if you use the `govuk_collection_radio_buttons` method there will be enough margin anyway so you don't need to add a `<br>` element.

It also looks like this file (and others) are missing a new line at the end (the red minus symbol) - it might be that you need to configure your editor to leave new lines at the bottom of files: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log